src/gui/executionlogwidget.h

```diff
@@ -30,33 +30,33 @@
 #define EXECUTIONLOGWIDGET_H
 
 #include <QWidget>
-#include "base/logger.h"
 
-class LogListWidget;
+#include "base/logger.h"
 
 namespace Ui
 {
     class ExecutionLogWidget;
 }
 
+class BaseLogModel;
+class LogFilterModel;
+class LogListView;
+
 class ExecutionLogWidget : public QWidget
 {
     Q_OBJECT
 
 public:
-    ExecutionLogWidget(QWidget *parent, const Log::MsgTypes &types);
-    void showMsgTypes(const Log::MsgTypes &types);
+    ExecutionLogWidget(QWidget *parent, Log::MsgTypes types);
```

Comment:
It would be nice to also switch the parameters order: `ExecutionLogWidget(Log::MsgTypes types, QWidget *parent);`

Reply:
done.